scripts/deploy-backend.sh

```diff
@@ -71,9 +70,19 @@ set_cf_envs()
         cf_cmd="cf unset-env $CGAPPNAME_BACKEND $var_name ${!var_name}"
         $cf_cmd
         continue
-    elif [[ ("$var_name" =~ "STAGING_") && ("$CF_SPACE" = "tanf-staging") ]]; then
-        sed_var_name=$(echo "$var_name" | sed -e 's@STAGING_@@g')
-        cf_cmd="cf set-env $CGAPPNAME_BACKEND $sed_var_name ${!var_name}"
+    # TODO: REMOVE DEV TESTING CODE
```

Comment:
since the SENDGRID_API_KEY is unique to each env and included in the circleci env vars, does this also need to be accounted for @elipe17 ?

Reply:
i think the dev testing code can be reverted before this merges.